.circleci/config.yml

```diff
@@ -30,10 +30,27 @@ jobs:
       - gem/setup-and-test:
           gemspec: silent_ping.gemspec
           bundler-version: 1.16.1
+  build-and-release:
+    working_directory: ~/repo
+    executor: gem/default
+    steps:
+      - gem/build:
+          gem-name: silent_ping
+      - gem/release:
+          gem-name: silent_ping
+          gem-credentials-env-name: $RUBYGEMS_API_KEY
 
 workflows:
-  test:
+  test-build-and-deploy:
     jobs:
       - test-with-ruby-latest
       - test-with-ruby-2-6-5
       - test-with-ruby-2-3-6
+      - build-and-release:
+          requires:
+          - test-with-ruby-latest
+          - test-with-ruby-2-6-5
+          - test-with-ruby-2-3-6
+          filters:
+            branches:
```

Comment:
im assuming that you are only going to run this on git tags, should this be `tags` instead of branches?

Reply:
Good catch! You are right, of course!